frontend/src/services/AxiosCustomize.tsx

```
@@ -0,0 +1,79 @@
+});
+
+instance.defaults.headers.common = { 'Authorization': `Bearer ${localStorage.getItem('access_token')}` }
+
+const handleRefreshToken = async () => {
+    // const res = await instance.get('/api/v1/auth/refresh');
+    // if (res && res.data) return res.data.access_token;
+    // else null;
+
+    return await mutex.runExclusive(async () => {
+        const res = await instance.get('/api/auth/refresh');
+        if (res && res.data) return res.data.access_token;
```

Biome (1.9.4)

[error] Change to an optional chain.

Unsafe fix: Change to an optional chain.


(lint/complexity/useOptionalChain)